diff --git a/index.js b/index.js
--- a/index.js
+++ b/index.js
@@ -1,6 +1,7 @@
 'use strict';
 const strictUriEncode = require('strict-uri-encode');
-const decodeComponent = require('decode-uri-component');
+const decodeModule = require('decode-uri-component');
+const decodeComponent = typeof decodeModule === 'function' ? decodeModule : decodeModule.default;
 const splitOnFirst = require('split-on-first');
 const filterObject = require('filter-obj');
 
